cabal.project

```diff
@@ -1,4 +1,4 @@
-index-state: 2020-05-15T00:00:00Z
+index-state: 2020-07-15T00:00:00Z
```

Comment:
Ok, that is brave. I have not been updating that to be any more recent than 2 weeks old.